fe/src/component/Tab/List.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const List = (props) => {
+  const onClickHandler = (event) => {
+    props.onSaveClickedID(event.target.id);
```

Comment:
```suggestion
const List = ({onSaveClickedID, id, className}) => {
  const onClickHandler = (event) => {
   onSaveClickedID(event.target.id);
```
인자를 destructuring해서 꺼내오면 props 키워드를 안써도 되겠네요!

https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#%EA%B0%9D%EC%B2%B4_%EA%B5%AC%EC%A1%B0_%EB%B6%84%ED%95%B4